web/assets/css/transmission-app.scss

```diff
@@ -553,7 +554,8 @@ $video-image: '../img/film.svg';
     .torrent-progress-details,
     .torrent-peer-details {
       color: var(--color-fg-primary);
-      font-size: small;
+      font-size: 0.875em;
+      font-weight: normal;
```

Comment:
This font-weight has no effect so should be safe to remove

I also want to make the font small again for progress and peer details.

Reply:
I've made that change, but i think normal should be the baseline. I don't see any reason for using small or x-small in this interface - at all.

Reply:
If you're fine with "small" and you don't see the issue with going bigger yet- doesn't mean it's not going to be an issue.